historianutils/historianutils.go

```diff
@@ -29,7 +29,7 @@ import (
 
 var (
 	// ServiceDumpRE is a regular expression to match the beginning of a service dump.
-	ServiceDumpRE = regexp.MustCompile(`^DUMP\s+OF\s+SERVICE\s+(?P<service>\S+):`)
+	ServiceDumpRE = regexp.MustCompile(`^DUMP\s+OF\s+SERVICE\s+(?P<service>.*):`)
```

Comment:
Can you add a comment that the tag <service> will include the optional priority of the service like HIGH or CRITICAL along with service name

Reply:
Sure, I will include that in the next commit.